### PR TITLE
Scenegraph layer website example

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: examples/website/scenegraph-layer/app.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        if (resp && resp.states) {
+          this.setState({data: resp.states});
+        }
+      });
+    //     window.requestAnimationFrame(this._animateData.bind(this));
+    //   });
+  }
+
+  _animateData() {
+    // const {data} = this.state;
```
Comment (ibgreen) [inline examples/website/scenegraph-layer/app.js]: Cut?

File: examples/website/scenegraph-layer/app.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+  _loadData() {
+    fetch(DATA_URL)
+      .then(resp => resp.json())
+      .then(resp => {
+        if (resp && resp.states) {
+          this.setState({data: resp.states});
+        }
+      });
+    //     window.requestAnimationFrame(this._animateData.bind(this));
```
Comment (ibgreen) [inline examples/website/scenegraph-layer/app.js]: Cut?

File: examples/website/scenegraph-layer/app.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+import {render} from 'react-dom';
+
+import {StaticMap} from 'react-map-gl';
+import DeckGL from '@deck.gl/react';
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+
+import {GLBScenegraphLoader, GLTFScenegraphLoader} from '@luma.gl/addons';
+import {registerLoaders} from '@loaders.gl/core';
+
+registerLoaders([GLBScenegraphLoader, GLTFScenegraphLoader]);
```
Comment (ibgreen) [inline examples/website/scenegraph-layer/app.js]: It would great if we could make these in a single loader. Do I just need to fix the multi-extension support in loaders.gl?

File: examples/webpack.config.local.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR, linkToLuma) {
     '@luma.gl/webgl-state-tracker': `${ROOT_DIR}/node_modules/@luma.gl/webgl-state-tracker`,
     '@luma.gl/webgl2-polyfill': `${ROOT_DIR}/node_modules/@luma.gl/webgl2-polyfill`,
     '@luma.gl/shadertools': `${ROOT_DIR}/node_modules/@luma.gl/shadertools`,
-    '@luma.gl/addons': `${ROOT_DIR}/node_modules/@luma.gl/addons`
+    '@loaders.gl/core': `${ROOT_DIR}/node_modules/@loaders.gl/core`
```
Comment (Pessimistress) [inline examples/webpack.config.local.js]: Why is this needed?

File: examples/website/scenegraph-layer/app.js [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+    this.state = {};
+
+    if (!window.demoLauncherActive) {
+      this._loadData();
+    }
+  }
+
+  componentWillUnmount() {
+    if (this._animationFrame) {
```
Comment (Pessimistress) [inline examples/website/scenegraph-layer/app.js]: What is this for?

File: examples/website/scenegraph-layer/style/app.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (Pessimistress) [inline examples/website/scenegraph-layer/style/app.css]: Is this file used?

**Comment by georgios-uber** [comment]
> I think we should make an effort to get an airplane model rather than a duck.

I showed this to some people and they all loved the duck 🦆 interface.

File: examples/webpack.config.local.js (hunk 4 above)
Comment (georgios-uber) [reply examples/webpack.config.local.js]: To ensure everyone is using the same loaders.gl/core.

File: examples/website/scenegraph-layer/app.js (hunk 5 above)
Comment (georgios-uber) [reply examples/website/scenegraph-layer/app.js]: Copied the whole file from another example. Forgot to delete this part.

File: examples/website/scenegraph-layer/style/app.css (hunk 6 above)
Comment (georgios-uber) [reply examples/website/scenegraph-layer/style/app.css]: Copied the whole file from another example. Not sure. Let me try to delete it and see what happens.

File: examples/website/scenegraph-layer/app.js (hunk 3 above)
Comment (georgios-uber) [reply examples/website/scenegraph-layer/app.js]: Yes.

File: examples/website/scenegraph-layer/style/app.css (hunk 6 above)
Comment (georgios-uber) [reply examples/website/scenegraph-layer/style/app.css]: Not needed, I will remove.
